replicatedhq/ship
# helm values not applied on ship update. Closes #370

What I Did
------------
Ability to include upstream changes to Helm's values.yaml file on updates.  Fixes #370.

How I Did it
------------
values.yaml file modified by the user is merged with the new version of values.yaml pulled from Helm repo.


How to verify it
------------
1. Run `ship init` to get a chart
1. Modify values (finish process to ensure state is saved)
1. Make changes multiple values in Helm repo (including the ones modified by user)
1. Run `ship update`
values.yaml file should include values set by user as well as values changed in the repo.


Description for the Changelog
------------
Helm values modified in the upstream repo are applied on updates.  Closes #370.


Picture of a Boat (not required but encouraged)
------------
![750x750bb](https://user-images.githubusercontent.com/1004892/44538713-03eeae80-a6b7-11e8-86aa-c813d281f25a.jpeg)
